app/javascript/packs/session-timeout-ping.ts

```diff
@@ -12,6 +12,7 @@ const frequency = parseInt(warningEl?.dataset.frequency || defaultTime, 10) * 10
 const warning = parseInt(warningEl?.dataset.warning || defaultTime, 10) * 1000;
 const start = parseInt(warningEl?.dataset.start || defaultTime, 10) * 1000;
 const timeoutUrl = warningEl?.dataset.timeoutUrl!;
+const sessionsApiUrl = warningEl?.dataset.sessionsApiUrl!;
```

Comment:
looks like this file escaped our uppercase URL pattern, but since we're here, could fix it?
```suggestion
const timeoutURL = warningEl?.dataset.timeoutUrl!;
const sessionsApiURL = warningEl?.dataset.sessionsApiUrl!;
```

Reply:
That second one is an interesting challenge, since I might expect `sessionsAPIURL` to be the more appropriate based on our rules? But also very strange to read 😬 (now I have more sympathy for the naming of [`XMLHttpRequest`](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest))

If it were me, I'd probably try to find a different name to avoid dealing with the weird casing 😂

Reply:
I took the lazy way out and went with `sessionsURL`